signer/local_cosigner.go

```diff
@@ -114,14 +117,15 @@ func (cosigner *LocalCosigner) SaveLastSignedState(signState SignStateConsensus)
 
 func NewLocalCosigner(cfg LocalCosignerConfig) *LocalCosigner {
```

Comment:
All usages of this function in the project should be updated to pass the "local" threshold ed25519 signature implementation (will be constant for now)